### DISCUSSION TITLE
Creating a Pull request for a specific commit

### DISCUSSION THREAD

**Discussion by chintan-wadhwa** [cafawo/FinancialDataAnalytics#23] [Q&A] [answered]
I wanted to make a correction in homework 2 of python but when I tried to create a pull request, it fetched all of my past commits instead of the latest one. Does anyone know how I can do that for a specific commit?

**Comment by cafawo** [comment, marked as answer]
Hi @chintan-wadhwa ,

It's great that you dive into pull requests!

I'd recommend the following workflow:
1. Create a new(!) clone of the origin repository, i.e. `git clone https://github.com/iwh-halle/FinancialDataAnalytics.git`
2. Make only the changes that you want to make to origin.
3. Commit and create a pull request. (Now you can delete the local copy of origin)

Best,
FW
